doc/dev/minikube.md

```diff
@@ -105,3 +105,56 @@ acp=$(kubectl get secret \
 # Use the password from the previous step when requested.
 cf auth -u admin -p "${acp}"
 ```
+
+### Advanced Topics
+
+#### Diego vs Eirini
+
+Diego is the standard scheduler used by kubecf to deploy CF
+applications. Eirini is an alternative talking more directly to the
```

Comment:
I added a comment regarding the same phrase above.

Reply:
Fixed with commit 9918b20dcb257d8865a289cee9653b710a4662a0.